src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    public static void getStatistic(String fromFileName, String toFileName) {
+        File myFile = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        List<Integer> listOfSupplyCosts = new ArrayList<>();
+        List<Integer> listOfBuyCosts = new ArrayList<>();
+        String result = calculateResult(fromFileName, listOfSupplyCosts,
+                listOfBuyCosts, stringBuilder);
+        createAndWriteToFile(toFileName, result);
+    }
+
+    public static String calculateResult(String fromFile, List<Integer> supplies,
+                                         List<Integer> buys, StringBuilder stringBuilder) {
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFile));
+            String string = bufferedReader.readLine();
```

Comment:
try with resources
go through checklist once more
this time be even more careful and attentive